test/unit/expressions.spec.ts

```diff
@@ -30,12 +30,7 @@ test.each(["1==1", "1===1", "1!=1", "1!==1", "1>1", "1>=1", "1<1", "1<=1", "1&&1
 );
 
 test.each(["'key' in obj", "'existingKey' in obj", "0 in obj", "9 in obj"])("Binary expression in (%p)", input => {
-    const tsHeader = "declare var obj: any;";
-    const tsSource = `return ${input}`;
-    const luaHeader = "obj = { existingKey = 1 }";
-    const result = util.transpileAndExecute(tsSource, undefined, luaHeader, tsHeader);
-
-    expect(result).toBe(eval(`let obj = { existingKey: 1 }; ${input}`));
+    util.testFunction(`let obj = { existingKey: 1 }; return ${input}`).expectToMatchJsResult();
```

Comment:
This would be nicer as a multi-line template string:
```ts
 util.testFunction`
    let obj = { existingKey: 1 };
    return ${input}
`.expectToMatchJsResult();
```